ui/app/AppLayouts/Profile/Sections/Ens/RegisterENSModal.qml

```diff
@@ -0,0 +1,239 @@
+                id: pvwTransaction
+                width: stack.width
+                fromAccount: selectFromAccount.selectedAccount
+                gas: {
+                    "value": gasSelector.selectedGasEthValue,
+                    "symbol": "ETH",
+                    "fiatValue": gasSelector.selectedGasFiatValue
+                }
+                toAccount: {
+                    return selectRecipient.selectedRecipient
+                }
+                asset: root.asset
+                currency: walletModel.defaultCurrency
+                amount: {
+                    const fiatValue = walletModel.getFiatValue(root.ensPrice || 0, root.asset.symbol, currency)
+                    return { "value": root.ensPrice, "fiatValue": fiatValue }
+                }
+                reset: function() {
+                    fromAccount = Qt.binding(function() { return selectFromAccount.selectedAccount })
+                    toAccount = Qt.binding(function() { return selectRecipient.selectedRecipient })
+                    asset = Qt.binding(function() { return root.asset })
+                    amount = Qt.binding(function() { return { "value": root.ensPrice, "fiatValue": walletModel.getFiatValue(root.ensPrice, root.asset.symbol, currency) } })
+                    gas = Qt.binding(function() {
+                        return {
+                            "value": gasSelector.selectedGasEthValue,
+                            "symbol": "ETH",
+                            "fiatValue": gasSelector.selectedGasFiatValue
+                        }
+                    })
+                }
+            }
+        }
+        TransactionFormGroup {
+            id: group4
+            headerText: qsTr("Send %1 %2").arg(Utils.stripTrailingZeros(root.ensPrice)).arg(root.asset.symbol)
+            footerText: qsTr("Sign with password")
+
+            TransactionSigner {
+                id: transactionSigner
+                width: stack.width
+                signingPhrase: walletModel.signingPhrase
+                reset: function() {
+                    signingPhrase = Qt.binding(function() { return walletModel.signingPhrase })
+                }
+            }
+        }
+    }
+
+    footer: Item {
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+        StyledButton {
+            id: btnBack
+            anchors.left: parent.left
+            //% "Back"
+            label: qsTrId("back")
+            onClicked: {
+                if (stack.isFirstGroup) {
+                    return root.close()
+                }
+                stack.back()
+            }
+        }
+        StyledButton {
+            id: btnNext
+            anchors.right: parent.right
+            label: qsTr("Next")
+            disabled: !stack.currentGroup.isValid
+            onClicked: {
+                const isValid = stack.currentGroup.validate()
+
+                if (stack.currentGroup.validate()) {
```

Comment:
Should check `isValid` and `isPending`, like so:
https://github.com/status-im/nim-status-client/blob/17aa8d274ed9c84cfe05284aa4ac175504e5008d/ui/app/AppLayouts/Wallet/SendModal.qml#L242-L243